doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -102,26 +102,24 @@ To enable Wi-Fi coexistence on the nRF70 Series device, complete the following s
    .. code-block::
 
       / {
-            nrf_radio_coex: nrf7002-coex {
-               status = "okay";
-               compatible = "nordic,nrf700x-coex";
-               req-gpios =     <&gpio0 24 (GPIO_ACTIVE_HIGH)>;
-               status0-gpios = <&gpio0 14 (GPIO_ACTIVE_HIGH)>;
-               grant-gpios =   <&gpio0 25 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
-         };
+            nrf_radio_coex: coex {
+                status = "okay";
+                compatible = "nordic,nrf7002-coex";
+                req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
+                status0-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+                grant-gpios = <&gpio0 24 (GPIO_PULL_DOWN | GPIO_ACTIVE_LOW)>;
+            };
       };
 
-#. Optionally, replace the node name ``nrf7002-coex`` with a custom one.
+#. Optionally, replace the node name ``coex`` with a custom one.
 #. Replace the pin numbers provided for each of the required properties:
 
    * ``req-gpios`` - GPIO characteristic of the device that controls the ``COEX_REQ`` signal of the nRF70 Series device.
    * ``status0-gpios`` - GPIO characteristic of the device that controls the ``COEX_STATUS0`` signal of the nRF70 Series device.
    * ``grant-gpios`` - GPIO characteristic of the device that controls the ``COEX_GRANT`` signal of the nRF70 Series device.
 
    .. note::
-      ``GPIO_PULL_UP`` is added to avoid a floating input pin and is required on some boards only.
-      If the target board is designed to avoid this signal being left floating, you can remove ``GPIO_PULL_UP`` to save power.
-
+      ``GPIO_PULL_DOWN`` is added to ensure a known state when the line is not driven and also grants SR when nRF70 Series device is powered off.
```

Comment:
add another bullet to explain `GPIO_ACTIVE_LOW` flag added to reflect the GRANT pin property